android/src/com/mapswithme/maps/search/HotelsFilter.java

```diff
@@ -0,0 +1,90 @@
+package com.mapswithme.maps.search;
+
+import android.support.annotation.NonNull;
+
+public class HotelsFilter
```

Comment:
No, I don't. To be honest, there is advice from google about "Avoid using enums, use the constants instead", because enums are the classes and can consume memory. But it maybe reasonable for last times, when devices had a 16 mb of RAM. Also, they suggest to use "Typedef Annotations" like a trade-off for using the "Enums". You can try to use it, but it's up to you - https://developer.android.com/studio/write/annotations.html (Typedef Annotations)